src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -357,18 +357,28 @@ std::string ContextAttribute::toJson(bool isLastElement)
 {
   std::string  out;
   bool         isNumber = false;
+  bool         isString = true;
 
   if (type == "number")
```

Comment:
Thus, the type for number is exactly "number" (I mean, "Number" is not a valid numeric type). It is fine... just to check that in our documentation we make clear that "number" is case-sensitive.